.misc/deploy.pypi.sh

```diff
@@ -11,11 +11,26 @@ echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+
+error=$(twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW" 2>&1 1>/dev/null)
```

Comment:
we still need to see the original output of twine.
It might have useful information.